lints/lint_rsa_public_exponent_not_in_range.go

```diff
@@ -27,12 +27,16 @@ func (l *rsaParsedTestsExpInRange) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rsaParsedTestsExpInRange) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	pubKey := c.PublicKey.(*rsa.PublicKey).E
+	key, ok := c.PublicKey.(*rsa.PublicKey)
+	if !ok {
+		return ResultStruct{Result: Error}, nil
+	}
+	exponent := key.E	
 	lowerBound := 65536 // 2^16 + 1
 	var upperBound big.Int
 	upperBound.Exp(big.NewInt(2), big.NewInt(256), nil)
```

Comment:
Why is this being run in every test? This should be run once during initialization and saved.